.github/workflows/nix.yml

```
@@ -16,10 +16,10 @@ jobs:
       matrix:
         channel:
           - nixos-unstable
-          - nixos-21.05
           - nixos-21.11
           - nixos-22.05
-    runs-on: ubuntu-20.04
+          - nixos-22.11
+    runs-on: ubuntu-22.04
```

This should be rater safe given that we use nix for the most part:

```suggestion
    runs-on: ubuntu-latest
```